flask_rest_paginate/__init__.py

```diff
@@ -132,8 +136,12 @@ def _get_page_link(page_number=None):
             except TypeError:
                 raise ValueError("Incorrect signature for hook")
 
-        return {
-            # TODO: use a better name for the pagination object
-            'pagination': OrderedDict(sorted(pagination_schema.items())),
-            'data': schema.dump(items, many=True) if marshmallow else f.marshal(page_obj.items, schema)
-        }
+        ret = {}
+        ret[self._data_object_key] = schema.dump(items, many=True) if marshmallow else f.marshal(page_obj.items, schema)
+        pagination_object = OrderedDict(sorted(pagination_schema.items()))
+        if self._pagination_object_key:
```

Comment:
I don't understand the purpose of this if condition as the `_pagination_object_key` is being set with a default value `pagination`. The condition here seems unnecessary.

Reply:
Hi @mtShaikh and thanks.
The purpose of this condition is to allow that pagination information could be moved directly in the root of the result object (istead of inside pagination key) by setting `_pagination_object_key` to null.
The initial value of `pagination` for that variable is to preserve the actual behaviour by default.

Reply:
Ahh i see...everything else looks okay
Could also please add an example about this functionality in the example directory and document it in the README as well?
Thanks

Reply:
Of course.... I'll do as soon as possible!

Reply:
@mtShaikh readme and examples updated.
Feel free to change any text if not clear enough.
Thanks